tests/random_test.py

```diff
@@ -476,6 +476,13 @@ def test_key_array_indexing_nd(self):
     self.assertRaisesRegex(IndexError, 'Too many indices.*',
                            lambda: keys[0, 1, None, 2])
 
+  def test_isinstance(self):
+    if not config.jax_enable_custom_prng:
+      self.skipTest("test requires config.jax_enable_custom_prng")
+    key = random.PRNGKey(0)
+    self.assertIsInstance(key, jnp.ndarray)
```

Comment:
Does this pass when jax_array is enabled? (JAX_ARRAY=1)

Maybe this change this to jax.Array?

Reply:
After this change yes. Before this change no.